index.md

```diff
@@ -17,8 +15,9 @@ However, this tool doesn't *require* using an OSTree based system. It works equa
 
 The toolbx environment is based on an [OCI](https://www.opencontainers.org/) image. On Fedora this is the `fedora-toolbox` image. This image is used to create a toolbox container that offers the interactive command line environment.
 
-Note that Toolbox makes no promise about security beyond what's already available in the usual command line environment on the host that everybody is familiar with.
+## Toolbox Security
```

Comment:
I don't think this is necessary.  The second paragraph is prominent enough.

I don't know why we need to keep talking about security so much.  Do we have a separate section on security for every program ever written? :)

Also, regardless of security, the description of a Toolbx environment is very important to convey what the project is about.  So, I don't think it's good to relegate it to a chapter on security.

Note that we deliberately don't use words like *unprivileged*; and now *container*, although one might say that containers aren't synonymous with heightened security.

Reply:
> I don't know why we need to keep talking about security so much.

I hope you are not serious. Or at least that's your personal opinion, and not a RedHat one. 

> Do we have a separate section on security for every program ever written? :)

I don't think it is appropriate to generalize security of container isolation with every program ever written. I am glad you are lucky to live in a country that are not targeted by attacks like this https://www.wired.com/story/developer-altered-open-source-software-to-wipe-files-in-russia/ but I am in Belarus and I don't have a job and money to move out.